data/sql_incremental_aggregates/prepare_schedule_a_aggregate_employer.sql

```diff
@@ -14,60 +17,18 @@ and is_individual(contb_receipt_amt, receipt_tp, line_num, memo_cd, memo_text, c
 group by cmte_id, cycle, employer
 ;
 
-alter table ofec_sched_a_aggregate_employer_tmp add column idx serial primary key;
+alter table ofec_sched_a_aggregate_employer_tmp add column idx serial;
+alter table ofec_sched_a_aggregate_employer_tmp add constraint uq_sched_a_aggregate_employer_tmp_cmte_id_cycle_employer unique (cmte_id, cycle, employer);
```

Comment:
One last question about this (and goes for all aggregate tables) - this constraint will be renamed with the indexes, right?  Or does that also have to be done separately to take care of the `_tmp` in it?

Reply:
We don't _need_ to adjust the `BaseModel` inheritance as part of this PR. We should do this soon in another PR in order to remove superfluous code.

Reply:
The constraint will be renamed with the rest of the indexes. The successful run of the unit tests confirms it as the `ON CONFLICT ON` clause uses the constraint name without the `_tmp`.